bucket/jd-cmd.json

```diff
@@ -1,16 +1,16 @@
 {
-    "version": "0.9.2",
+    "version": "1.0.0",
     "description": "Command line Java Decompiler.",
     "homepage": "https://github.com/kwart/jd-cmd/",
     "license": "GPL-3.0-only",
-    "url": "https://github.com/kwart/jd-cmd/releases/download/jd-cmd-0.9.2.Final/jd-cli-0.9.2-dist.zip",
-    "hash": "059c498e14510f588fede1f0230c6fb62e9e7ef34d8efd80ddee61a57d083404",
+    "url": "https://github.com/kwart/jd-cmd/releases/download/jd-cmd-1.0.0.Final/jd-cli-1.0.0.Final-dist.zip",
+    "hash": "7de21c63757a7cb1ccb44904eb318fa9ac16f3d49f7ff426c822516821ffc375",
     "bin": "jd-cli.jar",
     "checkver": {
         "github": "https://github.com/kwart/jd-cmd/",
         "regex": "jd-cmd-([\\d\\.]+)\\."
     },
     "autoupdate": {
-        "url": "https://github.com/kwart/jd-cmd/releases/download/jd-cmd-$version.Final/jd-cli-$version-dist.zip"
+        "url": "https://github.com/kwart/jd-cmd/releases/download/jd-cmd-$version.Final/jd-cli-$version.Final-dist.zip"
```

Comment:
"Final" missing from 0.9.2 appears to have been an outlier. Updating to pattern seen on 2/3:

![image](https://user-images.githubusercontent.com/4730164/68072469-8a435d80-fd5c-11e9-8760-5e5498e2112c.png)
![image](https://user-images.githubusercontent.com/4730164/68072474-90393e80-fd5c-11e9-80b4-3b9f42526467.png)